learntools/ml-insights/nbs/tut3.ipynb

```diff
@@ -0,0 +1,215 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**[Machine Learning for Insights Home Page](kaggle.com/learn/ml-insights).**\n",
+    "\n",
+    "---\n",
+    "\n",
+    "# Partial Dependence Plots\n",
+    "\n",
+    "While feature importance shows what variables most affect predictions, partial dependence plots give us more detail about how a feature affects predictions.\n",
+    "\n",
+    "This is useful to answer questions like:\n",
+    "\n",
+    "* Controlling for all other house features, what impact do longitude and latitude have on home prices? To restate this, how would similarly sized houses be priced in different areas.\n",
+    "\n",
+    "* Are predicted health differences between two groups due to differences in their diets, or due to some other factor?\n",
+    "\n",
+    "If you are familiar with linear or logistic regression models, partial dependence plots can be interepreted similarly to the coefficients in those models.  Though, compared to those model coefficients, partial dependence plots can capture far more complex patterns from your data.  If you aren't familiar with linear or logistic regressions, don't worry about this comparison.\n",
+    "\n",
+    "We will show a couple examples, explain the interpretation of these plots, and then review the code to create these plots.\n",
+    "\n",
+    "# How it Works\n",
+    "\n",
+    "Like permutation importance, **partial dependence plots are calculated after a model has been fit.**  The model is fit on real data that has not been artificially manipulated in any way.  \n",
+    "\n",
+    "In our soccer example, teams may differ in many ways. How many passes they made, shots they took, goals they scored, etc. At first glance, it may seem difficult to disentangle the effect of these features. Because the teams that made many passes may also take more shots and score more goals in most cases. \n",
+    "\n",
+    "To see how partial plots separate out the effect of each featuer, we start by considering a single row of data. For example, that row of data might represent a team that had the ball 50% of the time, made 100 passes, took 10 shots and scored 1 goal.\n",
+    "\n",
+    "We will use the fitted model to predict our outcome (probability their player won \"man of the game\". But we **repeatedly alter the value for one variable** to make a series of predictions.  We could predict the outcome if the team had the ball only 40% of the time. We then predict with them having the ball 50% of the time.  Then predict again for 60%.  And so on.  We trace out predicted outcomes (on the vertical axis) as we move from small values of ball possession to large values (on the horizontal axis).\n",
```

Comment:
It would be great if this matched the first example plot (which currently deals with 'Goals Scored', rather than % of time with ball).